### PR TITLE
Update dlp-configure-endpoint-settings.md

### PR DESCRIPTION
**Pull request by kurtsarens**
Better use direct language. The wording was confusing and could be interpreted that files directly under the folder are also excluded. Confirmed with page in Dutch language

### CODE REVIEW

File: microsoft-365/compliance/dlp-configure-endpoint-settings.md
Comment (JohanFreelancer9) [inline microsoft-365/compliance/dlp-configure-endpoint-settings.md]:
```suggestion
- Valid file path that ends with `\*`, which means only files under subfolders. Files directly under the folder are not excluded. <br/>For example: `C:\Temp\*`
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kurtsarens, kindly commit the suggested change. Many thanks.

**Comment by kurtsarens** [comment]
@JohanFreelancer9 who has write access?

**Comment by JohanFreelancer9** [comment]
@kurtsarens, as the creator of this PR, you should be able to commit these changes.

**Comment by kurtsarens** [comment]
@JohanFreelancer9 I have 2 commits... what else is needed here?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@kurtsarens, thank you for the commits. Nothing else is required from you. After approval, your changes will be merged and will be live.

**Comment by yogkumgit** [comment]
@chrfox Please review and merge this PR. Thanks

**Comment by chrfox** [comment]
holding this pending feedback from PT

**Pull request merged**
